config/webpack.config.dev.js

```diff
@@ -227,11 +227,9 @@ module.exports = {
 		}),
 		// Make NODE_ENV environment variable available to the JS code, for example:
 		// if (process.env.NODE_ENV === 'development') { ... }.
-		new DefinePlugin({
-			'process.env': {
-				NODE_ENV: '"development"'
-			}
-		}),
+		new DefinePlugin({'process.env.NODE_ENV': JSON.stringify('development')}),
+		// Inject prefixed environment variables within code, when used
+		new EnvironmentPlugin(Object.keys(process.env).filter(/^REACT_APP_/.test)),
```

Comment:
This is failing for me on node 7.8 and 8.11.

> ERROR: Method RegExp.prototype.test called on incompatible receiver undefined

`.filter(RegExp.prototype.test.bind(/^REACT_APP_/))` seems to work but is less readable than `.filter(key => /^REACT_APP_/.test(key))`